templates/user/profile.html

```diff
@@ -27,205 +27,53 @@
         onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
     </figure>
     <div>
-      <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
-      <p name=about>{{ user.about|htmlnewline or 'Introduce yourself here' }}</p>
+      <h1 name=username>{{ user.username }}</h1>
+      <p name=about>{{ user.about|htmlnewline }}</p>
```

Comment:
We need these defaults in order to be able to type in

Reply:
as you made this comment on your own code I'm not sure about what I have to do with this 😛

Reply:
as you made this comment on your own code I'm not sure about what I have to do with this 😛